custom_components/tahoma/water_heater_devices/domestic_hot_water_production.py

```diff
@@ -11,14 +11,20 @@
 
 from ..entity import OverkizEntity
 
-STATE_MANUAL = "manual"
-STATE_AUTO = "auto"
+DHWP_TYPE_MURAL = "io:AtlanticDomesticHotWaterProductionV2_MURAL_IOComponent"
+DHWP_TYPE_CE_FLAT_C2 = "io:AtlanticDomesticHotWaterProductionV2_CE_FLAT_C2_IOComponent"
+DHWP_TYPE_MBL = "modbuslink:AtlanticDomesticHotWaterProductionMBLComponent"
```

Comment:
Do you plan to move this to our lib?

Reply:
you mean pyoverkiz? Not sure. This relates to HA specific commands doesn't it?

Reply:
That’s specific to overkiz, like all the others  state or command we already moved.

Reply:
@tetienne for now, we don't have a dict of controllable names in our library. I think it will be hard to maintain, so I would be okay to have them here for now.